package.json

```diff
@@ -84,7 +84,7 @@
     "jasmine-check": "0.1.5",
     "jest": "23.6.0",
     "marked": "0.3.19",
-    "microtime": "2.1.8",
+    "microtime": "3.0.0",
```

Comment:
microtime 2.x do not support node 12 (https://github.com/wadey/node-microtime/issues/59)